pymongo/encryption_options.py

```diff
@@ -219,3 +219,44 @@ def __init__(
         # Maps KMS provider name to a SSLContext.
         self._kms_ssl_contexts = _parse_kms_tls_options(kms_tls_options)
         self._bypass_query_analysis = bypass_query_analysis
+
+
+class RangeOpts:
+    """Options to configure encrypted queries using the rangePreview algorithm."""
+
+    def __init__(
+        self,
+        sparsity: int,
+        min: Optional[Any] = None,
+        max: Optional[Any] = None,
+        precision: Optional[int] = None,
+    ) -> None:
+        """Options to configure encrypted queries using the rangePreview algorithm.
+
+        .. note:: Support for Range queries is in beta.
+           Backwards-breaking changes may be made before the final release.
+
+        :Parameters:
+          - `sparsity`: An integer.
+          - `min`: A BSON scalar value corresponding to the type being queried.
+          - `max`: A BSON scalar value corresponding to the type being queried.
+          - `precision`: An integer, may only be set for double or decimal128 types.
+
+        .. versionadded:: 4.4
+        """
+        self.min = min
+        self.max = max
+        self.sparsity = sparsity
+        self.precision = precision
+
+    def as_doc(self):
```

Comment:
ReadConcern, WriteConcern, and Collection all use the "document" property for this:
```
    @property
    def document(self) -> Dict[str, Any]:
```
Can we use that same pattern here?